docker/docker.go

```diff
@@ -217,23 +217,23 @@ func IsContainerRunning(ctx context.Context, dClient *client.Client, hostname st
 	return false, nil
 }
 
-func localImageExists(ctx context.Context, dClient *client.Client, hostname string, containerImage string) (bool, error) {
+func localImageExists(ctx context.Context, dClient *client.Client, hostname string, containerImage string) error {
 	var err error
 	for i := 1; i <= RetryCount; i++ {
 		logrus.Infof("Checking if image [%s] exists on host [%s], try #%d", containerImage, hostname, i)
 		_, _, err = dClient.ImageInspectWithRaw(ctx, containerImage)
 		if err != nil {
 			if client.IsErrNotFound(err) {
-				logrus.Infof("Image [%s] does not exist on host [%s]: %v", containerImage, hostname, err)
-				return false, nil
+				logrus.Debugf("Image [%s] does not exist on host [%s]: %v", containerImage, hostname, err)
+				return err
 			}
-			logrus.Warnf("Error checking if image [%s] exists on host [%s]: %v", containerImage, hostname, err)
+			logrus.Debugf("Error checking if image [%s] exists on host [%s]: %v", containerImage, hostname, err)
 			continue
 		}
 		logrus.Infof("Image [%s] exists on host [%s]", containerImage, hostname)
-		return true, nil
+		return nil
 	}
-	return false, fmt.Errorf("Error checking if image [%s] exists on host [%s]: %v", containerImage, hostname, err)
+	return fmt.Errorf("error checking if image [%s] exists on host [%s]: %v", containerImage, hostname, err)
```

Comment:
Why is this `error` and the rest is `Error`?

Reply:
@superseb we should fix the rest. It should align with the go best practices where the error should be in lower case